db/pgstorage/pgstorage.go

```diff
@@ -305,6 +322,30 @@ func (p *PostgresStorage) GetDeposit(ctx context.Context, depositCounterUser uin
 	return &deposit, err
 }
 
+// GetGERByDepositCnt gets the oldest global exit root which the deposit count is greater than the given value.
+func (p *PostgresStorage) GetGERByDepositCnt(ctx context.Context, networkID uint8, depositCnt uint, dbTx pgx.Tx) (*etherman.GlobalExitRoot, error) {
+	var (
+		ger                      etherman.GlobalExitRoot
+		exitRoots                [][]byte
+		getGERByLocalExitRootSQL string
+	)
+	if networkID == MainNetworkID {
+		getGERByLocalExitRootSQL = "SELECT e.block_id, e.global_exit_root, e.exit_roots FROM syncv2.exit_root AS e INNER JOIN mtv2.root AS r ON e.exit_roots[$1] = r.root WHERE r.deposit_cnt > $2 AND e.block_id = 0 ORDER BY r.deposit_cnt ASC LIMIT 1"
```

Comment:
Why `e.block_id = 0` instead of `e.block_id > 0`?

Reply:
Because the query needs to look for the global exit roots from the trusted state